docs/en-US/component/tree.md

```diff
@@ -123,43 +123,43 @@ tree/draggable
+| Name                  | Description                                                                                                                                                                                                                                                                                                                                                                 | Type                                                   | Default |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ | ------- |
+| data                  | tree data                                                                                                                                                                                                                                                                                                                                                                   | ^[Array]                                               | —       |
+| empty-text            | text displayed when data is void                                                                                                                                                                                                                                                                                                                                            | ^[string]                                              | —       |
+| node-key              | unique identity key name for nodes, its value should be unique across the whole tree                                                                                                                                                                                                                                                                                        | ^[string]                                              | —       |
+| props                 | configuration options, see the following table                                                                                                                                                                                                                                                                                                                              | ^[object]`{children: string, label: string, disabled}` | —       |
```

Comment:
The type description here is incomplete. How about link the type to the following props table?

Reply:
Agree.